custom-words.txt

```diff
@@ -94,3 +93,4 @@ YubiKeys
 # Forbidden words
 !auto-fill
 !auto-filled
+Rickabaugh
```

Comment:
❓ Did lint need this? Your only usage is in a link which wasn't checked I thought.

We keep this alphabetically sorted by the way.